index.html

```diff
@@ -257,10 +257,10 @@ <h3>Ecosystem Overview</h3>
 <a>verifiable presentation</a> that proves the holder possesses the required verifiable credentials with certain characteristics.
 Examples of verifiers include employers, security personnel, and websites.
         </dd>
-        <dt><a>identifier registry</a></dt>
+        <dt><a>Verifiable data registry</a></dt>
         <dd>
 A role a system may perform by mediating the creation and verification of <a>issuer</a> identifiers,
-keys and other relevant data required to use verifiable credentials. Some configurations may require correlatable
+keys and other relevant data like schemas, revocation registries, etc. required to use verifiable credentials. Some configurations may require correlatable
```

Comment:
Concrete suggestion: Change "schemas" to "data validation schemas"

Reasoning: The term "schema" is too broad and has led to confusion in the group.